src/SourceGenerators/Uno.UI.SourceGenerators/DependencyObject/DependencyObjectGenerator.cs

```diff
@@ -455,31 +456,31 @@ private void WriteAttachToWindow(INamedTypeSymbol typeSymbol, IndentedStringBuil
 					WriteRegisterLoadActions(typeSymbol, builder);
 
 					builder.AppendMultiLineIndented($@"
```

Comment:
Same as above and in the rest of the PR, could we use a raw multiline string literal with `$$` to avoid the `{{`-s?